core/chains/evm/bulletprooftxmanager/eth_broadcaster.go

```diff
@@ -345,20 +359,36 @@ func (eb *EthBroadcaster) handleInProgressEthTx(etx EthTx, attempt EthTxAttempt,
 	}
 	parentCtx := context.TODO()
 
-	if etx.Simulate {
-		simulationCtx, cancel := context.WithTimeout(parentCtx, SimulationTimeout)
-		defer cancel()
-		if b, err := simulateTransaction(simulationCtx, eb.ethClient, attempt, etx); err != nil {
-			if jErr := evmclient.ExtractRPCError(err); jErr != nil {
-				eb.logger.CriticalW("Transaction reverted during simulation", "ethTxAttemptID", attempt.ID, "txHash", attempt.Hash, "err", err, "rpcErr", jErr.String(), "returnValue", b.String())
-				etx.Error = null.StringFrom(fmt.Sprintf("transaction reverted during simulation: %s", jErr.String()))
-				return eb.saveFatallyErroredTransaction(&etx)
-			}
-			eb.logger.Warnw("Transaction simulation failed, will attempt to send anyway", "ethTxAttemptID", attempt.ID, "txHash", attempt.Hash, "err", err, "returnValue", b.String())
-		} else {
-			eb.logger.Debugw("Transaction simulation succeeded", "ethTxAttemptID", attempt.ID, "txHash", attempt.Hash, "returnValue", b.String())
-		}
+	checkerSpec, err := etx.GetChecker()
+	if err != nil {
+		return errors.Wrap(err, "parsing transmit checker")
```

Comment:
Nit: error message should be "failed to parse transmit checker". I don't think we have guidelines on error messages but would be nice to make them uniform in the future in structure / grammar.

Reply:
I personally prefer this format for brevity reasons; it's more readable once you have a long stack trace of errors. Not sure there's official guidance here but I generally like uber's style guide a lot: https://github.com/uber-go/guide/blob/master/style.md#error-wrapping

Definitely open to changing it if folks disagree though

Reply:
I have no opinion about the string as long as it is unique in the codebase since really the only reason to wrap errors is so you can find them.